docs/core/integration-quality-scale/rules/runtime-data.md

```diff
@@ -15,14 +15,13 @@ Because of the added typing, we can use tooling to avoid typing mistakes.
 
 ## Example implementation
 
-The type of a `ConfigEntry` can be extended with the type of the data put in `runtime_data`.
+The type of a `ConfigEntry` should be extended with the type of the data put in `runtime_data`.
```

Comment:
Typing is not in the scope at the point this is being done (that is part of a higher tier).

I therefore, don't agree with adding that pressure onto it in this stage.

Reply:
> Typing is not in the scope at the point this is being done (that is part of a higher tier).
> 
> I therefore, don't agree with adding that pressure onto it in this stage.

Should this then be a separate rule (in which tier)?
Or should it be a sub-rule for `strict-typing` and only apply to `platinum`?
Or should we simply ignore it?

Reply:
> Should this then be a separate rule (in which tier)?

I'm not sure why? Strict typing already handles this case.